### ISSUE TITLE
Seeing Error: Method "web_app_set_bottom_bar_color" is unsupported in Mini Apps version 7.6

### ISSUE THREAD

**Issue by alexandrzavalii** [Telegram-Mini-Apps/nextjs-template#24]
Anyone seen this before? just followed the instructions and seeing this error

<img width="397" alt="Screenshot 2024-11-19 at 23 13 44" src="https://github.com/user-attachments/assets/97d494b6-6db2-481a-bd2d-0e2f1d6d9506">

**Comment by a0983627353** [comment]
> Anyone seen this before? just followed the instructions and seeing this error
> 
> <img width="397" alt="Screenshot 2024-11-19 at 23 13 44" src="https://github.com/user-attachments/assets/97d494b6-6db2-481a-bd2d-0e2f1d6d9506">
>

**Comment by heyqbnk** [comment]
Yes, because this method is unsupported in version 7.6. Should probably be fine now
